target/xilinx/xilinx_ips/common.mk

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Cyril Koenig <cykoenig@iis.ee.ethz.ch>
+
+all: load-artifacts $(PROJECT).xpr save-artifacts
+
+# Build IP
+xlnx_%.xpr:
+	$(vivado_env) $(VIVADO) -mode batch -source tcl/run.tcl
+
+save-artifacts:
+
+load-artifacts:
```

Comment:
This was to avoid override the target `all` as done [before](https://github.com/pulp-platform/carfield/blob/main/target/xilinx/xilinx/common.mk#L18).
If it's bad practice I can remove it